src/doc/trpl/comments.md

```diff
@@ -1,47 +1,45 @@
 % Comments
 
-Now that we have some functions, it's a good idea to learn about comments.
+Now that we have some functions, it’s a good idea to learn about comments.
```

Comment:
I don't plan on doing a mega-edit to update them, it's only because I'm already doing a readthrough of all this that I'm fixing it.